### ISSUE TITLE
enhacement: 640x400 resolution

### ISSUE THREAD

**Issue by jsmolina** [viti95/FastDoom#81]
Tartar doom or doom mbf has this higher resolution, like in windows versions.
https://gitlab.com/ludicrous_peridot/cod10mbf

As many of us would like to run fastdoom on powerful machines, We'd like to have higher resolutions available, if posible.

**Comment by dougvj** [comment]
I wanted this too so I started on this last week. Here's my WIP


![image](https://user-images.githubusercontent.com/6045454/203180811-008fdb9d-18c3-4399-8432-09c2e973306f.png)

Once it's working fully i'll submit a PR.

**Comment by jsmolina** [comment]
> I wanted this too so I started on this last weekend. Here's my WIP
> 
> ![image](https://user-images.githubusercontent.com/6045454/203180811-008fdb9d-18c3-4399-8432-09c2e973306f.png)
> 
> Once it's working fully i'll submit a PR.

no luck? hehe

**Comment by dougvj** [comment]
I got really far, I have most of the rendering correct, but the UI and the weapon are still a struggle:

![image](https://github.com/viti95/FastDoom/assets/6045454/8c322931-14f3-4f80-b0be-c5e79ae50065)


I also know that some of the changes broke other modes even though I tried to properly guard everything in defs.

I also have to rebase against master. 

I work on this every once in a while in my free time, so it's slow going, but I am happy to commit and push what I have if it's useful.
